setup.cfg

```diff
@@ -18,4 +18,7 @@ log_cli_level = WARNING
 
 [flake8]
 max-line-length = 88
-ignore = W503, E121, E126, E211, E225, E501, E203, E402, F401, F811, E231
+ignore = W503, E121, E126, E211, E225, E501, E203, E402, F401, F811, E231,
```

Comment:
probably out of scope, but it would be very useful to document these codes as well with comments

Reply:
🤔 but flake8 does this as well. Maybe simply a link to their doc?

Reply:
added links